tmt/steps/provision/artemis.py

```diff
@@ -550,6 +557,7 @@ def go(self) -> None:
             arch=self.get('arch'),
             image=self.get('image'),
             hardware=self.get('hardware'),
+            kickstart=self.get('kickstart'),
```

Comment:
This will not work, since no version of Artemis recognizes this key, even if it'd be `null`. No change to the Artemis plugin is needed, yet.

Reply:
Alright, I'll remove all changes related to Artemis